tests/integrations/virtual_environment/test_VirtualEnvironment.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import MagicMock
+
+from briefcase.integrations.virtual_environment import VenvEnvironment
+
+
+def test_recreate_true(mock_tools, dummy_console, tmp_path):
+    """Test context manager recreates environment when recreate=True."""
```

Comment:
A minor note - We're in a test, so the fact that it's testing is implied. 
```suggestion
    """Context manager recreates environment when recreate=True."""
```

Reply:
This wasn't intended as a once-off comment - it's a general comment about test descriptions throughout this, and other files.